### ISSUE TITLE
http_requests_total is only available as a default metric

### ISSUE THREAD

**Issue by jpslopes** [trallnag/prometheus-fastapi-instrumentator#15]
Hello, I noticed that the default metrics contain the metric `http_requests_total`. As this metric is only defined inside the method `default`, it was necessary to create it as a custom metric:
```python
def http_requests_total(metric_namespace='', metric_subsystem='') -> Callable[[Info], None]:
    total = Counter(
        name="http_requests_total",
        documentation="Total number of requests by method, status and handler.",
        labelnames=(
            "method",
            "status",
            "handler",
        ),
        namespace=metric_namespace,
        subsystem=metric_subsystem,
    )

    def instrumentation(info: Info) -> None:
        total.labels(info.method, info.modified_status, info.modified_handler).inc()

    return instrumentation
```
It would be great to have this metric available as a method like `latency` and `response_size`.

Thanks!

**Comment by trallnag** [comment]
Hey @jpslopes, yes I think this is a good improvement. Will integrate it this weekend

**Comment by trallnag** [comment]
Implemented in [5.5.0](https://github.com/trallnag/prometheus-fastapi-instrumentator/releases/tag/releases%2F5.5.0).

Docs <https://trallnag.github.io/prometheus-fastapi-instrumentator/metrics.html#prometheus_fastapi_instrumentator.metrics.requests>

**Comment by jpslopes** [comment]
Thanks.
